packages/core-components/src/hooks/useSupportConfig.ts

```diff
@@ -23,7 +23,7 @@ export type SupportItemLink = {
 
 export type SupportItem = {
   title: string;
-  icon?: string;
+  icon?: string | JSX.Element;
```

Comment:
This wouldn't match the type in the config as it only is possible to pass in strings.

Reply:
oh, you can configure this from yaml? I was unaware

Now sure how to resolve that - strings still work from yaml, so existing behaviour wouldn't change - but if using the component in react you'd have more flexibility